generated-docs/out/backends/summary.md

```diff
@@ -19,29 +19,30 @@ Each backend has two type parameters:
-``AkkaHttpBackend``                  ``scala.concurrent.Future``      ``akka.stream.scaladsl.Source[ByteString, Any]``  yes (regular & streaming)  yes
-``AsyncHttpClientFutureBackend``     ``scala.concurrent.Future``      n/a                                               yes (regular)              no
-``AsyncHttpClientScalazBackend``     ``scalaz.concurrent.Task``       n/a                                               yes (regular)              no
-``AsyncHttpClientZioBackend``        ``zio.Task``                     ``zio.stream.Stream[Throwable, Byte]``            yes (regular & streaming)  no
-``AsyncHttpClientMonixBackend``      ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  no
-``AsyncHttpClientCatsBackend``       ``F[_]: cats.effect.Concurrent`` n/a                                               no                         no
-``AsyncHttpClientFs2Backend``        ``F[_]: cats.effect.Concurrent`` ``fs2.Stream[F, Byte]``                           yes (regular & streaming)  no
-``OkHttpSyncBackend``                None (``Identity``)              n/a                                               yes (regular)              no
-``OkHttpFutureBackend``              ``scala.concurrent.Future``      n/a                                               yes (regular)              no
-``OkHttpMonixBackend``               ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  no
-``Http4sBackend``                    ``F[_]: cats.effect.Effect``     ``fs2.Stream[F, Byte]``                           no                         no
-``HttpClientSyncBackend``            None (``Identity``)              n/a                                               no                         no
-``HttpClientFutureBackend``          ``scala.concurrent.Future``      n/a                                               yes (regular)              no
-``HttpClientMonixBackend``           ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  yes
-``HttpClientFs2Backend``             ``F[_]: cats.effect.Concurrent`` ``fs2.Stream[F, Byte]``                           yes (regular & streaming)  yes
-``HttpClientZioBackend``             ``zio.Task``                     ``zio.stream.Stream[Throwable, Byte]``            yes (regular & streaming)  yes
-``FinagleBackend``                   ``com.twitter.util.Future``      n/a                                               no                         no
-==================================== ================================ ================================================= ========================== ===================
+==================================== ================================ ================================================= ========================== =================== ===============================
```

Comment:
you need to modify the `summary.md` in `docs` - this is the directory with the generated source (after processing by mdoc). So this will be overwritten on the next release.

Also, I would add the "support host overrides" note as a side-note for particular backends, or a side-note somewhere later in this page. This is a summary table which aims to help people choose the right one. I doubt supporting host overrides would be a decisive factor most of the time :)

Reply:
I moved this info to specific backend sections